src/database.c

```diff
@@ -184,6 +187,29 @@ static void abort_transaction(MDB_txn *txn)
 	dbi_init = 0;
 }
 
+/*
+ * Convert path to a hash value. Used when the path exceeds the LMDB key limit(511).
+ * Note: Returned value must be deallocated.
+ */
+char *path_to_hash(const char *path)
```

Comment:
Also, any reason why this can't be static?

Reply:
You are right